### ISSUE TITLE
Update-Help gives "Failed to update Help for the module(s) 'PSScaffold'" 

### ISSUE THREAD

**Issue by yooakim** [RedeployAB/PSScaffold#2]
When I run UPdate-Help and have PSScaffold installed I get the following error:

```
C:\Users\yooak> update-help
update-help : Failed to update Help for the module(s) 'PSScaffold' with UI culture(s) {en-US} : The value of the HelpInfoUri key in the module manifest must resolve to a container or root URL on a website where the help files are stored. The HelpInfoUri 'https://github.com/RedeployAB/PSScaffold#usage' does not resolve to a container.
At line:1 char:1
+ update-help
+ ~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (:) [Update-Help], Exception
+ FullyQualifiedErrorId : InvalidHelpInfoUri,Microsoft.PowerShell.Commands.UpdateHelpCommand
```
I think the URL is wrong... `https://github.com/RedeployAB/PSScaffold`might be the correct one?

**Comment by KarlGW** [comment]
@yooakim I'll look into this as soon as possible. Sorry for the inconvenience.

**Comment by KarlGW** [comment]
@yooakim it has been addressed in the newest version 1.1.5. Due to the nature of Update-Help, it will try to get the help for older versions. So to stop getting errors I would suggests to remove the versions of the module older that 1.1.5 after updating it.
